src/Platform/Microsoft.Testing.Platform/UI/FileUtilities.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace Microsoft.Testing.Platform.UI;
```

Comment:
helpers namespace/folder.

Reply:
can this please wait once we connect it to dotnet test?

Reply:
As far as I can see, you did it, so resolving.